### PR TITLE
Housekeeping release for 0.2.0

### PR DESCRIPTION
**Pull request by mdenker**
This is a housekeeping PR to get ready for the 0.2.0 release.

### CODE REVIEW

File: doc/source/authors.rst [hunk 1]
```diff
@@ -20,6 +20,8 @@ contribution, and may not be the current affiliation of a contributor.
 * Sonja Grün (orcid: 0000-0003-2829-2220) [1, 2]
 * Pier Stanislao Paolucci (orcid: 0000-0003-1937-6086) [3]
 * Michael Denker (orcid: 0000-0003-1255-7300) [1]
+* Cosimo Lupo (orcid: 0000-0002-2651-1277) [3,4]
+* Frederico Marmoreo (orcid: 0000-0002-5933-2873) [3]
```
Comment (cosimolupo) [inline doc/source/authors.rst:24]:
```suggestion
* Federico Marmoreo (orcid: 0000-0002-5933-2873) [3]
```

File: .zenodo.json [hunk 2]
```diff
@@ -2,25 +2,37 @@
...
+    {
+      "orcid": "0000-0002-2651-1277",
+      "affiliation": "Istituto Nazionale di Fisica Nucleare (INFN), Sezione di Roma, Rome, Italy",
+      "name": "Lupo, Cosimo",
+      "role": "Project member"
+    },
+    {
+      "orcid": "0000-0002-5933-2873",
+      "affiliation": "Istituto Nazionale di Fisica Nucleare (INFN), Sezione di Roma, Rome, Italy",
+      "name": "Marmoreo, Frederico",
```
Comment (cosimolupo) [inline .zenodo.json:18]:
```suggestion
      "name": "Marmoreo, Federico",
```

File: doc/source/authors.rst [hunk 3]
```diff
@@ -20,6 +20,8 @@ contribution, and may not be the current affiliation of a contributor.
 * Sonja Grün (orcid: 0000-0003-2829-2220) [1, 2]
 * Pier Stanislao Paolucci (orcid: 0000-0003-1937-6086) [3]
 * Michael Denker (orcid: 0000-0003-1255-7300) [1]
+* Cosimo Lupo (orcid: 0000-0002-2651-1277) [3,4]
```
Comment (cosimolupo) [inline doc/source/authors.rst:23]:
```suggestion
* Cosimo Lupo (orcid: 0000-0002-2651-1277) [3]
```

File: codemeta.json [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+        },
+        {
+            "id": "http://orcid.org/0000-0002-5933-2873",
+            "type": "Person",
+            "affiliation": {
+                "type": "Organization",
+                "name": "Istituto Nazionale di Fisica Nucleare (INFN), Sezione di Roma, Rome, Italy"
+            },
+            "familyName": "Marmoreo",
+            "givenName": "Frederico"
```
Comment (cosimolupo) [inline codemeta.json:34]:
```suggestion
            "givenName": "Federico"
```

**Review by rgutzen — APPROVED** [review]

**Comment by cosimolupo** [comment]
@mdenker PR #77 should also be added in the release notes file, when talking about the update of the README file

**Pull request merged**